Published/asc051.md

```diff
@@ -32,7 +32,7 @@ In addition to `Float:Concentration`, the standard Amount unit can be added as f
 |------------|--------------|------------|
 | molar      | M            | 1          |
 | millimolar | mM           | 1e-3       |
-| micromolar | uM           | 1e-6       |
+| micromolar | μM           | 1e-6       |
```

Comment:
Wondering whether we should include the abbreviation field here. As far as I know, this is neither defined in the spec nor is it declared in ASCs for other units.

Reply:
I see. Yea, that is indeed a new field. My initial thought process in review was that this is purely indicative and not prescriptive (i.e. it will not be part of the spec). 

I'm down for adding a line to explain that or to remove this table

Reply:
@ahadik would you be fine with us removing this table? I'm ambivalent about this, but Rhys does bring up a good point about keeping things consistent for now, and having a separate ASC for defining abbreviation/multipliers

Reply:
decided to remove it for now